buku

```diff
@@ -2186,12 +2189,13 @@ class BukuDb:
                 LOGERR('No matching index %s', index)
                 return False
 
+            single_record = len(results) == 1
```

Comment:
bugfix: when invoking with a _set of indices_, JSON output only printed 1 record before